src/components/AddBook.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const AddBook = () => (
+  <div>
+    <input type="text" placeholder="new book" />
```

Comment:
- [ ] Please render a second input to accept the author of the book so we can comply with the project requirements.